lib/src/controllers/player_controller.dart

```diff
@@ -31,8 +33,11 @@ class PlayerController extends ChangeNotifier {
   /// Provides [max] duration of currently provided audio file.
   int get maxDuration => _maxDuration;
 
+  final ValueKey<String> _playerKey =
+      ValueKey("${Isolate.current.hashCode}${Random().nextInt(10000)}");
```

Comment:
Since audio waveforms don't make use of isolates, the hashcode will same for every player now as you are taking random int between 0-10000, this still doesn't make sure that the key will be the same. Because two random int values can be the same.

Reply:
It is okay now as expected.  Previously, the pub.dev version that used UniqueKey().toString() for the player key resulted in duplicate keys in the iOS release mode. 

<img width="1080" alt="Screenshot 2024-07-02 at 17 05 22" src="https://github.com/SimformSolutionsPvtLtd/audio_waveforms/assets/108823930/8dfd2709-327f-4566-8a22-f5834c4e601d">


The current shortHash version is working correctly, as I expected.